ios/airship_flutter/Package.swift

```diff
@@ -0,0 +1,26 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "airship_flutter",
+    platforms: [.macOS(.v10_15), .iOS(.v14), .tvOS(.v14), .visionOS(.v1)],
+    products: [
+        .library( name: "airship-flutter", targets: ["airship_flutter"])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/urbanairship/airship-mobile-framework-proxy.git", from: "11.2.2")
```

Comment:
need to set the correct new version of the proxy once https://github.com/urbanairship/airship-mobile-framework-proxy/pull/75 is merged and released